documentation/asciidoc/topics/proc_disabling-fips-mode-cr.adoc

```diff
@@ -0,0 +1,33 @@
+:_module-type: PROCEDURE
+
+[id="proc_disabling-fips-mode-cr_{context}"]
+= Disabling FIPS mode in your {brandname} CR
+
+[role="_abstract"]
+For {brandname} {ispn_version}, an issue exists when you want your {brandname} instance to run on any {openshiftshort} cluster that has Federal Information Processing Standards (FIPS) mode enabled. {brandname} {ispn_version} does not support FIPS mode, so you must disable FIPS mode in your {brandname} CR.
```

Comment:
This seems more like a RN text. I would not phrase it this way
Few more points:
{ispn_version} resolves to the full Infinispan version for example, 13.0.10.Final-redhat-00001
For downstream look for the attributes in the rhdg-attributes.adoc that is in the downtream repo
But in this sentence the version number would have to be hardcoded
One sentence per line :)

Reply:
Can I use {ProductVersion}, because this content is specifically for downstream?